lib/google/gax/settings.rb

```diff
@@ -400,9 +400,11 @@ def upper_camel_to_lower_underscore(string)
     #   {
     #     "interfaces": {
     #       "google.fake.v1.ServiceName": {
-    #         "retry_codes": {
-    #           "idempotent": ["UNAVAILABLE", "DEADLINE_EXCEEDED"],
-    #           "non_idempotent": []
+    #         "retry_codes_def": {
```

Comment:
It seems odd to me to have `retry_codes_def` nested under `retry_codes`. There isn't a `_def` node nested under `retry_params`, and there isn't a `_def` node nested under `methods`. Why is `retry_codes` different?